### PR TITLE
ARROW-15847: [Python][CI] Ensure we have a nightly Python build with parquet encryption disabled

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
Follow up on https://github.com/apache/arrow/pull/12565#issuecomment-1061762076

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
@github-actions crossbow submit test-conda-python-3.8-pandas-latest test-conda-python-3.8-pandas-nightly

**Comment by jorisvandenbossche** [comment]
OK, now it's correctly skipping the tests in the one build (while still running them in the other build)

**Review by pitrou — APPROVED** [review]

**Comment by pitrou** [comment]
Thank you @jorisvandenbossche !

**Comment by pitrou** [comment]
@github-actions crossbow submit test-conda-python-3.8-pandas-latest test-conda-python-3.8-pandas-nightly

File: docker-compose.yml [hunk 1]
```diff
@@ -950,6 +950,7 @@ services:
     shm_size: *shm-size
     environment:
       <<: *ccache
+      PARQUET_REQUIRE_ENCRYPTION:  # inherit
```
Comment (jorisvandenbossche) [inline docker-compose.yml:953]: Nice, didn't know that worked as well!

**Pull request closed without merging**